docs/source/developer/event_api.rst

```diff
@@ -0,0 +1,5 @@
+Event API
+=========
+
+.. automodule:: numba.core.event
+    :members:
```

Comment:
I like this approach

Reply:
We should do this more. Less duplication

Reply:
Agree